src/mmw/js/src/core/views.js

```diff
@@ -673,6 +679,19 @@ var MapView = Marionette.ItemView.extend({
             }
         });
     },
+
+    renderFocusBounds: function() {
```

Comment:
This feature will need to be re-worked to support displaying a collection of bounding boxes and points instead of a single feature.